### PR TITLE
Add github test action.

### PR DESCRIPTION
**Pull request by jguiditta**
Minor changes to configs to remove warnings, and take
first step towards modernizing structure, which will be
fully done in another patch.

Signed-off-by: Jason Guiditta <jguiditt@redhat.com>

### CODE REVIEW

**Comment by jguiditta** [comment]
Example test run - https://github.com/jguiditta/koji_wrapper/actions/runs/1287291044

**Review by jpichon — APPROVED** [review]
LGTM.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+name: Run tests
+on:
+  push:
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+jobs:
+    test:
+      runs-on: ubuntu-latest
```
Comment (fuzzball81) [inline .github/workflows/test.yml:9]: Just curious can we use fedora here or do we have to use ubuntu?

**Review by fuzzball81 — APPROVED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (jguiditta) [reply .github/workflows/test.yml:9]: github runners are all ubuntu afaik, so unless we want to host our own, I believe this is the only option.  If we find there is a different way later, I am not opposed.

Comment (jguiditta) [reply .github/workflows/test.yml:9]: Just to point to something specific, here are the supported github runners:
https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources

**Pull request merged**